src/app/tests/suites/certification/Test_TC_DD_3_7.yaml

```diff
@@ -14,25 +14,130 @@
 # Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
 
 name:
-    3.3.7. [TC-DD-3.7] Commissioning Flow - Concurrent - Negative scenario [DUT
+    3.3.7. [TC-DD-3.7] Commissioning Flow - Concurrent - Negative Scenario [DUT
     - Commissioner]
-
 config:
-    nodeId: 0x12344321
+    nodeId: "0x12344321"
     cluster: "Basic"
     endpoint: 0
-
 tests:
-    - label:
-          "Repeat steps 1.a-1.o from [TC-DD-3.5]. Force Fail any step between
-          1.a and 1.j."
+    - label: "Test Step"
       verification: |
+          Verification Steps
+      disabled: true
+    - label:
+          "Commissioner has regulatory and fabric information available and has
+          accurate date, time and timezone"
+      verification: |4+
 
       disabled: true
+    - label:
+          "Commissioner and Commissionee discover each other and connect via the
+          discovery mode applicable for the DUT."
+      verification: |4+
 
+      disabled: true
     - label:
```

Comment:
Why is this "disabled" grouped with the next test?  There is a lot of that in here.

Reply:
i will  check and update this script